src/lib/containers/table/SynapseTable.tsx

```diff
@@ -846,6 +846,7 @@ export default class SynapseTable extends React.Component<
           const isLockedFacetColumn =
             column.name.toLowerCase() === lockedFacet?.facet?.toLowerCase() // used in details page to disable filter the column
           const isEntityIDColumn =
+            columnModel &&
```

Comment:
This is the bug fix.  Verify that we found a columnModel before looking at the CM name and type.